papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -53,16 +63,18 @@
         return ResponseEntity.ok(sprints);
     }
 
-    public ResponseEntity<SprintDto> addSprint(Project project, @Valid SprintDto sprintDto) {
+    public ResponseEntity<SprintDto> addSprint(@Valid SprintDto sprintDto, Long projectId) {
         LocalDateTimePeriod enrollmentPeriod = localDateTimePeriodMapper.mapFromApi(sprintDto.getEnrollmentPeriod());
         LocalDateTimePeriod durationPeriod = localDateTimePeriodMapper.mapFromApi(sprintDto.getDurationPeriod());
```

Comment:
Walidacja admina faktycznie ma większy sens przed sprawdzaniem poprawności dat - te dwa `LocalDateTimePeriod` można dać niżej przed `isPreviousSprintOpen` w sumie

Reply:
Rozbijanie aż tak deklaracji i przypisania zmiennych chyba nie ma sensu, dość nieładnie to wygląda i ciężej czyta

Reply:
Spoko, ja bym po prostu dała deklarację bliżej miejsca użycia, ale obojętne

Reply:
Ok, to zostawiam tak jak jest i zamykam